oop_bank_account/razi.py

```diff
@@ -0,0 +1,28 @@
+
+class Account :
+    def __init__(self, name, balance):
+        self.AccOwner=name
+        self.balance =balance
+
+    def deposit(self,amount):
+        self.balance = self. balance + amount
```

Comment:
This won't work since `self` and `balance` should be concatenate together: `self.balance` not `self. balance`